### ISSUE TITLE
Alternative text of images

### ISSUE THREAD

**Issue by Czirion** [carpentries-lab/metagenomics-analysis#44]
**Reviewer's comments**:
1)The image https://carpentries-incubator.github.io/shell-metagenomics/fig/02-01-01.png does not have a sufficiently detailed alt text.
2)The image https://carpentries-incubator.github.io/shell-metagenomics/fig/02-02-01.svg does not have a sufficiently detailed alt text.
3)The alt text of image https://carpentries-incubator.github.io/shell-metagenomics/fig/02-05-01.png could be slightly more descriptive.
4)The image https://user-images.githubusercontent.com/67386612/118720027-ba433c00-b7ee-11eb-87e5-7496fde5763e.png should have a more detailed alt text (that being said, the Figure 1 caption should be sufficient for this)
5)The alt text of image https://carpentries-incubator.github.io/metagenomics/fig/03-02-03.png has some phrasing issues, and could also be enhanced.
6)The alt text of image https://carpentries-incubator.github.io/metagenomics/fig/03-03-01.png could be enriched with more information.
7)There is insufficient alt text in the image https://carpentries-incubator.github.io/metagenomics/fig/03-04-01.png and https://carpentries-incubator.github.io/metagenomics/fig/03-05-01.png
8)There is insufficient alt text in the image https://carpentries-incubator.github.io/metagenomics/fig/03-06-01.png

**To do**: Improve alternative text in the mentioned images following the alternative text guidelines: [UCSF: Accessibile Images Best Practices](https://it.ucsf.edu/how-to/accessible-images-best-practices)
[The bIg Hack: Avoid these common alt-text mistakes](https://bighack.org/avoid-these-mistakes-when-writing-alt-text-descriptions-for-images/). 
And review the alternative text in the rest of the images not mentioned here.

**Comment by Czirion** [comment]
1) done
2) done
3) done
4) done
5)done

**Comment by Czirion** [comment]
6) done
7)done
8)done
